extensions/@shopgate-product-reviews/frontend/RatingStars/index.jsx

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { hot } from 'react-hot-loader';
 import appConfig from '@shopgate/pwa-common/helpers/config';
+import { useWidgetSettings } from '@shopgate/engage/core/hooks/useWidgetSettings';
```

Comment:
you could also import 'useWidgetSettings' from '@shopgate/engage/core', because there's an index file which exports among other things all hooks.